org.multimc.MultiMC.metainfo.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.multimc.MultiMC</id>
+  <launchable type="desktop-id">org.multimc.MultiMC.desktop</launchable>
+  <name>MultiMC</name>
+  <summary>Manage Minecraft instances with ease</summary>
+  <description>
+    <p>Overview</p>
+    <p>MultiMC is a free, open source launcher for Minecraft. It allows you to have multiple, cleanly separated instances of Minecraft (each with their own mods, texture packs, saves, etc) and helps you manage them and their associated options with a simple and powerful interface.</p>
+    <p>Features</p>
+    <ul>
+      <li>Manage multiple instances of Minecraft at once</li>
+      <li>Start Minecraft with a custom resolution</li>
+      <li>Change Java's runtime options (including memory options)</li>
+      <li>Shows Minecraft's console output in a colour coded window</li>
+      <li>Kill Minecraft easily if it crashes / freezes</li>
+      <li>Custom icons and groups for instances</li>
+      <li>Forge integration (automatic installation, version downloads, mod management)</li>
+      <li>Minecraft world management</li>
+      <li>Import and export Minecraft instances to share them with anyone</li>
+      <li>Supports every version of Minecraft that the vanilla launcher does</li>
+    </ul>
+    <p>NOTE: This distribution is not verified by, affiliated with, or supported by the MultiMC team.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://multimc.org/images/screenshots/main.png</image>
+      <image type="source">https://multimc.org/images/screenshots/editmods.png</image>
+      <image type="source">https://multimc.org/images/screenshots/version.png</image>
+      <image type="source">https://multimc.org/images/screenshots/console.png</image>
+      <image type="source">https://multimc.org/images/screenshots/settings.png</image>
+    </screenshot>
```

Comment:
A screenshot can only have one image tag.

Reply:
So you need multiple screenshot tag here

Reply:
I think the image tags should look better now.